CP04OSSM/CP04OSSM_D00013_ingest.csv

```diff
@@ -29,6 +29,4 @@ mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00013/cg_data/dcl11/metbk/*.metbk.log,CP04OSSM-SBD11-06-METBKA000,telemetered,Available,
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00013/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,telemetered,Available,
 mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00013/cg_data/dcl12/hyd*/*hyd*.log,CP04OSSM-SBD12-03-HYDGN0000,telemetered,Available,
-mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00013/cg_data/dcl12/pco2a/*.pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,telemetered,Available,
-mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00013/cg_data/dcl11/fdchp2/*fdchp2.log,CP04OSSM-SBD11-08-FDCHPB000,telemetered,Available,
-mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00013/cg_data/dcl12/fdchp1/*fdchp1.log,CP04OSSM-SBD12-08-FDCHPA000,telemetered,Available,
```

Comment:
Looks good.  Test data should not be ingested.

Reply:
looks good